### PR TITLE
Publish GitHub action workflow

### PR DESCRIPTION
**Pull request by yhy-1**
Will need to have NPM registry token in setting-> action secret. Environment secrets name is NPM_Token

### CODE REVIEW

File: .github/workflows/manual-publishing.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    needs: pre-publish-checks
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          registry-url: "https://registry.npmjs.org"
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```
Comment (mellis481) [inline .github/workflows/manual-publishing.yml:38]: All secrets will have to be set by Dan to be available.

File: .npmrc [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#registry.npmjs.org/:_authToken=${NODE_AUTH_TOKEN}
+registry=https://registry.npmjs.org/
+always-auth=true
+legacy-peer-deps=true
```
Comment (mellis481) [inline .npmrc:4]: @dgreene1 Are you cool w/ this?

File: .github/workflows/manual-publishing.yml (hunk 1 above)
Comment (dgreene1) [inline .github/workflows/manual-publishing.yml:38]: I’d rather not store this as a secret. Let’s consider using np instead.

**Comment by dgreene1** [comment]
I spoke to @yhy-1 & @mellis481 and we’re going to use https://github.com/sindresorhus/np instead of this workflow since np is an established part of the NPM community and will help with standardization.

**Pull request closed without merging**